Lokasenna_GUI v2/Library/Classes/Class - TextEditor.lua

```diff
@@ -6,7 +6,7 @@
     https://github.com/jalovatt/Lokasenna_GUI/wiki/TextEditor
 
     Creation parameters:
-	name, z, x, y, w, h[, text, caption, pad]
+	name, z, x, y, w, h[, text, caption, pad, disableEditing]
```

Comment:
This comment implies that disableEditing can be supplied as a positional parameter, but that won't be the case unless you also:

- modify the constructor to take this parameter
- add a line to the constructor to do `txt.disableEditing = txt.disableEditing or disableEditing`

Reply:
oh, i interpreted that as "anything in the brackets is optional"

Reply:
There are some parameters that are "Optional", and then more under "Additional parameters":

https://github.com/jalovatt/Lokasenna_GUI/wiki/2.00-TextEditor#additional-parameters

Both sets are optional, but only the ones in the constructor parameter list can be passed as positional parameters.